frame_id: "base_scan"
scan_topic_name: "/scan"
end_frame_id: "map"

cluster:
  marker_topic_name: "/cluster_visualization_marker_array"
  marker_color:
    r: 1
    g: 0
    b: 0
    a: 0.5
  eps: 0.25
  min_samples: 10
  id: 0

subcluster:
  marker_topic_name: "/subcluster_visualization_marker_array"
  marker_color:
    r: 0
    g: 0
    b: 1
    a: 1
  eps: 0.05
  min_samples: 3
  id: 0

valid_clusters:
  marker_topic_name: "/valid_clusters"
  marker_color:
    r: 1
    g: 1
    b: 0
    a: 1
  id: 0

parking_pose:
  marker_topic_name: "/parking_goal"

marker_timelife: 0.5

max_point_distance: 3

scan_angle:
  min: 0
  max: 6.28

wall_filter:
  work: 0
  ransac_epsilon: 0.2
  distance_threshold: 0.2

simple_pattern:
  pattern_width: 0.6
  pattern_offset: 0.2
  required_subclusters: 5

distance_from_centroid: 0.3

pose_buffer:
  max_samples: 4
  confirm_threshold: 2
  distance_threshold: 0.1

